platform/ke02/export/platform/irq.hpp

```diff
@@ -0,0 +1,71 @@
+#ifndef KE02_PLATFORM_IRQ_HPP_
```

Comment:
I suggest to implement this module - it can be done using `NVIC_*` set of functions and `__enable_irq(),  __disable_irq()` functions